.gitignore

```diff
@@ -1,4 +1,26 @@
-node_modules
-dist
-debug.log
```

Comment:
Please add this debug.log too, sometimes it magically appears...

Reply:
Got it!